code/src/components/ThoughtsCard.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import moment from "moment";
+
+//ThoughtsCard component needs a lot of data that was gathered in App component when
+//doing the main Fetch to get the initial thoughts array, so we get it passed here via props
+export const ThoughtsCard = ({ id, message, timeCreated, hearts, addLike }) => {
+  //handleLikes is the function that will add a heart to the specific thought in the API
+  //by doing a POST request
+  //using that thought's id we make sure we add the heart to the right thought
+  const handleLikes = () => {
+    //Stretch goal: created a click counter so we know how many times the Heart button
+    //has been clicked and send that data in the callback function to App component
+    let clicks = 0;
+    clicks += 1;
+    fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${id}/like`, {
+      method: "POST",
+      body: "",
```

Comment:
While executing POST request for `likes` amount, we don't actually have to implement `body` property inside options object. API expects from us only unique ID inside url, that's the essence of this request.

Instead of writing :

```jsx
fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${id}/like`, {
    method: 'POST',
    body: {},
    headers: { 'Content-Type': 'application/json' }
});
```

We can just write :

```jsx
fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${id}/like`, {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' }
});
```